cloud/google/cmd/gce-machine-controller/README.md

```diff
@@ -0,0 +1,27 @@
+# GCE Machine Controller
+
+The GCE Machine Controller is a machine controller implementation for clusters running on Google Compute Engine.
+
+## Development
+The instructions below will enable you to get started with running your own version of the gce-machine-controller on a GCE Kubernetes cluster.
+
+### Prerequisites
+
+1. Follow the "Before you begin" section from the google cloud container registry [pushing and pulling images documentation](https://cloud.google.com/container-registry/docs/pushing-and-pulling).
+1. From this directory, run `make dev_push`.
+1. Note from the ouput of the above step the name of the image. It should be in this format, `gcr.io/my-gcp-project-name/gce-machine-controller:0.0.7-dev`.
+1. From the same terminal in which you will create your cluster using gcp-deployer, run `export MACHINE_CONTROLLER_IMAGE=gcr.io/my-gcp-project-name/gce-machine-controller:0.0.7-dev`. Note that the value should be equal to the name of the image you noted in the output from the previous step.
+1. Follow the steps listed at [gcp-deployer](../../../../gcp-deployer/) and create a new cluster.
+
+### Running a Custom GCE Machine Controller
+
+1. Make a change to gce-machine-controller. For example, edit main.go and insert the following print statement `glog.Error("Hello World!")` below `logs.InitLogs()`.
+1. From this folder, run `make dev_push`.
+1. Run `kubectl patch deployment clusterapi -p "{\"spec\":{\"template\":{\"metadata\":{\"labels\":{\"date\":\"`date +'%s'`\"}}}}}"`.
+
+### Verifying Your Environment
+
+1. Note the name of the master instance from the `gcp-deployer create` output, the default is `gce-master-test1`. 
```

Comment:
Would it make more sense to get at the logs via kubectl than sshing in and using docker?

Reply:
Yes, and using kubectl is way better / more awesome. I updated with some changes after I moved to kubectl.